public_html/wp-content/themes/pattern-directory/src/components/search/index.js

```diff
@@ -3,34 +3,36 @@
  */
 import { __ } from '@wordpress/i18n';
 import { Icon, search } from '@wordpress/icons';
+import { useInstanceId } from '@wordpress/compose';
 
-const CategorySearch = ( { defaultValue, isLoading, isVisible, onUpdate, onSubmit } ) => {
+const Search = ( { defaultValue, isLoading, isVisible, onUpdate, onSubmit } ) => {
+	const instanceId = useInstanceId( Search );
 	if ( isLoading ) {
-		return <span className="category-search--is-loading" />;
+		return <span className="search is-loading" />;
```

Comment:
I feel like the class `search` will have some old rules interfering when we get to .org since it's a common class name.

Reply:
That's fair - I was back-and-forth about whether it should be `pattern-search` as the class name, but I don't think we've been consistent about that (requiring `pattern-` a prefix for component classNames). What do you think?

Reply:
Yeah, we haven't been consistent. I'm good with moving to a `pattern` prefix or maybe just `wporg`?

Reply:
We've at least been somewhat consistent with `pattern`, so I'll switch to that and maybe write up a short doc we can refer to.